invokeai/app/services/image_storage.py

```diff
@@ -77,27 +77,30 @@ def get(self, image_type: ImageType, image_name: str) -> Image:
         if cache_item:
             return cache_item
 
-        image = Image.open(image_path)
+        image = PIL.Image.open(image_path)
         self.__set_cache(image_path, image)
         return image
 
     # TODO: make this a bit more flexible for e.g. cloud storage
     def get_path(self, image_type: ImageType, image_name: str) -> str:
-        path = os.path.join(self.__output_folder, image_type, image_name)
+        if image_type == ImageType.LOCAL:
```

Comment:
The UI uses the `uploads` folder for this sort of thing. Maybe it would be fine to have a CLI command (not a node) to "upload" a file (and maybe create a load_image node to load it from the uploads folder)? It _would_ result in a file copy, but images aren't that big, and then we'd be following the same pattern as the UI (which also means you could later load the graph in the UI and it would work).